app/forms/AssessmentFormProvider.scala

```diff
@@ -24,14 +24,17 @@ import javax.inject.Inject
 
 class AssessmentFormProvider @Inject() extends Mappings {
 
-  def apply(exemptionIds: Seq[String]): Form[AssessmentAnswer] =
+  def apply(exemptionIds: Seq[String], exemptionCount: Int = 1): Form[AssessmentAnswer] = {
+    val messagesKey = if (exemptionCount == 1) { "assessment.error.required.onlyOne" }
+    else { "assessment.error.required" }
     Form(
       "value" ->
-        text("assessment.error.required")
+        text(messagesKey)
           .verifying(
-            "assessment.error.required",
-            value => exemptionIds.contains(value) || value == AssessmentAnswer.NoExemption.toString
+            messagesKey,
+            value => value == "true" || value == AssessmentAnswer.NoExemption.toString
```

Comment:
do we still need this bit

|| value == AssessmentAnswer.NoExemption.toString

Reply:
Not really! :) Refactored